### ISSUE TITLE
Add API feature to support basic backup flows

### ISSUE THREAD

**Issue by avive** [spacemeshos/api#19]
The Testnet rewards program is on hold until we can build a backup agent api client that will backup all account balances, transactions, counters and smeshing rewards. We need to make sure that the API supports such backup workflows and prioritize implementing it in API 0.2.
@lrettig - we need to focus on this and come up with methods design to support full meta-mesh backups.

**Comment by avive** [comment]
What we would like to enable is an API client that can create a basic backup of the current global state at a specific layer with the data and indexes:
1. All accounts with non-zero balance. (k: account id, value: counter, balance}, enum of all accounts.
2. The rewards assigned to each account. L: account id, value: [rewards]

Note that this info is meta-mesh - e.g. not on the mesh itself. The data is all output of the STF of that layer. See the data design epic for more info on this topic.

**Comment by lrettig** [comment]
Seems pretty straightforward. Do we already have endpoints that can do this in #16 or do we need new ones?

**Comment by avive** [comment]
We might, we just need to develop a workflow for backing the meta-mesh of a network for a specific layer a recoverable / searchable way and make sure we have specified the api methods needed to support it.

**Comment by lrettig** [comment]
Yup. That part probably doesn't belong in the "api" repository, though -
we'll probably want a separate utility to do this. (Or maybe it could be
part of spacecraft?)

>
